iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalTableOperations.java

```diff
@@ -224,7 +227,8 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
           InternalCatalogMetricsConstant.METADATA_UPDATE_LATENCY);
 
       houseTable = houseTableMapper.toHouseTable(updatedMetadata);
-
+      // Set the storage type of table in house table before persisting in hts
+      houseTable.setStorageType(fileIOManager.getStorage(io()).getType().getValue());
```

Comment:
I assume this io() in turn will use the storage selector for new tables when they are first committed?

Reply:
➕ lets use `fileIO` instance instead of `io()`

Reply:
done